src/utils/db.ts

```diff
@@ -2,27 +2,27 @@ import { Connection, createConnection } from 'typeorm';
 
 import { PostModel } from '../models/PostModel';
 import { UserModel } from '../models/UserModel';
-import UserSession from '../models/UserSessionModel';
+import { UserSessionModel } from '../models/UserSessionModel';
 
-const models =[
+const models = [
   UserModel,
   PostModel,
-  UserSession
+  UserSessionModel
 ];
 
 export default async function resellConnection(): Promise<Connection> {
   return await createConnection({
     database: process.env.DB_NAME,
     entities: models,
     host: process.env.DB_HOST,
-    logging: false, //set to true to help with debugging
+    logging: (process.env.LOGGING?.toLowerCase() === "true"), //set to true to help with debugging
```

Comment:
in the future, it would be best to refactor these environment variable checks to be booleans in a constants/utils file so we don't need to extract like this everytime

Reply:
I agree. Will work on that after MVP launch :) Thx for the feedback